exasol/toolbox/tools/dependencies_check.py

```diff
@@ -0,0 +1,35 @@
+from tomlkit import loads
+from sys import exit
+import pytest
```

Comment:
I would prefer top level imports and IMHO `pytest` is not required. 

```suggestion
import tomlkit
import sys 
```